### DISCUSSION TITLE
Why not add sponsor block feature and auto popup video feature

### DISCUSSION THREAD

**Discussion by Elanngo-Madheswaran** [TeamNewPipe/NewPipe#10725] [Questions] [answered]
Why you not adding sponsor block feature which labels every part of the video like intro ,outro and we set them to skip automatically when they come to those parts. And another feature that is when we exit the app while some video is playing it automatically goes to popup mode

**Comment by AudricV** [comment, marked as answer]
Sponsor blocking support is currently rejected, see https://newpipe.net/blog/pinned/newpipe-and-online-advertising/.

Opening popup player when leaving the app is already supported, look at Settings, Video and audio, Minimize on app switch and select the appropriate action.
